drag_and_drop_v2/public/css/drag_and_drop_edit.css

```diff
@@ -315,8 +320,9 @@
 
 .xblock--drag-and-drop--editor .remove-item .icon.remove {
     background: #fff;
+    color: #0072a7;
 }
 .xblock--drag-and-drop--editor .remove-item .icon.remove:before,
 .xblock--drag-and-drop--editor .remove-item .icon.remove:after {
-    background: #2e83cd;
+    background: #236299;
```

Comment:
What you did makes sense, but I think I would prefer to keep the number of different colors the same as before this patch - all colors that used to be `#2e83cd` should be changed to the same new color, for example - if that's possible. It might not be possible if you had to brighten it in some cases and darken in others.